rootfs/scheduler/__init__.py

```diff
@@ -870,9 +884,9 @@ def _wait_until_pods_terminate(self, namespace, labels, current, desired):
 
             time.sleep(1)
 
-        logger.debug("{} pods in namespace {} are terminated".format(delta, namespace))  # noqa
+        logger.debug("{} pods in namespace {} are terminated".format(delta, namespace))
```

Comment:
flake8, yeah - That used to be a line that was too long (over 99 chars). No more. The CI runs flake8 and co so you'd see a red build otherwise